lib/handler/attachersHandler.js

```diff
@@ -12,7 +12,7 @@ export default {
     const nextElements = [];
 
     const attachedOutgoing = (element.attachers || [])
-      .map(att => att.outgoing.reverse())
+      .map(att => (att.outgoing ? att.outgoing.reverse() : []))
```

Comment:
Null object pattern, again:

```suggestion
      .map(attacher => (attacher.outgoing || []).reverse())
```